web/viewer.css

```diff
@@ -722,18 +757,30 @@ html[dir="rtl"] .splitToolbarButton > .toolbarButton {
 .toolbarButton,
 .secondaryToolbarButton,
 .overlayButton {
-  border: 0 none;
+  border: var(--overlay-button-border);
   background: none;
   width: 28px;
   height: 28px;
 }
 .overlayButton {
   background-color: var(--overlay-button-bg-color);
+  border-radius: 2px;
+  color: var(--main-color);
+  font-size: 12px;
+  line-height: 14px;
+  user-select: none;
+  cursor: default;
+  box-sizing: border-box;
```

Comment:
Is it really necessary to duplicate these rules here, i.e. why [remove this line](https://github.com/mozilla/pdf.js/pull/13203/files#diff-d1047e1376717fbe7c554bb385cd48b2dcbbb135e02884be3db97ad072ba741eL800)?